packages/api-v2/src/student/student.controller.ts

```diff
@@ -71,6 +72,31 @@ export class StudentController {
     return student;
   }
 
+  @UseGuards(JwtAuthGuard)
+  @Patch("me/onboard")
+  async onBoard(
+    @Req() req: AuthenticatedRequest,
+    @Body() updateStudentDto: OnboardStudentDto
+  ): Promise<UpdateStudentResponse> {
+    const uuid = req.user.uuid;
+    const updateResult = await this.studentService.update(uuid, {
+      ...updateStudentDto,
```

Comment:
@aryanshah701 will the validation pipe remove extra fields from the dto? if not, we should explicitly pull the named fields we want from this, to avoid implicitly storing random stuff/overwriting fields in our db.